frontend/src/components/PhoneNumberInput/PhoneNumberInput.tsx

```diff
@@ -79,231 +59,49 @@ export interface PhoneNumberInputProps
    * `libphonenumber-js/examples.mobile.json` if none provided.
    */
   examples?: { [country in CountryCode]: NationalNumber }
+
+  /**
+   * Whether international phone numbers are allowed. Defaults to `true`.
+   * If allowed, the phone number input will be prefixed with the country code,
+   * and the selected country will be displayed in the input's left add-on, and
+   * autoformatting will be enabled.
+   */
+  isAllowInternational?: boolean
```

Comment:
/supernit: `allowSomething` is already a `true|false` naming. The `is` prefix may not have been needed for the variable name.